src/GafferScene/SceneAlgo.cpp

```diff
@@ -832,6 +917,53 @@ SceneAlgo::AttributeHistory::Ptr SceneAlgo::attributeHistory( const SceneAlgo::H
 	return result;
 }
 
+SceneAlgo::OptionHistory::Ptr SceneAlgo::optionHistory( const SceneAlgo::History *optionsHistory, const IECore::InternedString &option )
+{
+	ScenePlug::GlobalScope globalScope( optionsHistory->context.get() );
```

Comment:
I think the `GlobalScope` is unnecessary here, because the captured `optionsHistory->context` should have been from a process that was sanitised in the first place. So I think just `Context::Scope` should be sufficient.